Package.swift

```diff
@@ -1,9 +1,15 @@
 import PackageDescription
 
 let package = Package(
-  name: "SnowMan",
-  dependencies: [
-    .Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,7))
-  ]
+  	name: "SnowMan",
+  	targets: [
+  		Target(name: "Dictionary"),
+  		Target(name: "SnowManLogic", dependencies: ["Dictionary"]),
+ 		Target(name: "Application", dependencies: ["SnowManLogic"]),
+ 		Target(name: "Specs", dependencies: ["SnowManLogic"]),
+  	],
+  	dependencies: [
+    	.Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,8))
```

Comment:
Tabs and spaces probably work differently in your text editor. Plus even if you don't see the problem before you push you should at least be looking at your PR before you have us review it, that way you can catch it.